src/components/binding/binding.js

```diff
@@ -150,13 +162,16 @@ genotet.BindingView = function(viewName, params) {
     .on('genotet.gene', function(event, data) {
       this.data.tracks[data.trackIndex].fileName = data.fileName;
       this.loader.loadFullTrack(data.trackIndex, data.fileName,
-        this.data.chr);
+        this.data.chr, false);
     }.bind(this));
 
   $(this.loader)
     .on('genotet.chr', function(event, chr) {
       this.panel.updateChr(chr);
     }.bind(this))
+    .on('genotet.addTrack', function() {
+      this.loader.loadBindingList();
```

Comment:
Firstly, update binding list when update is necessary. Because user may upload files while using, and we should keep the binding list in panel is always updated.

Secondly, panel should be updated after the binding list is loaded. If this is requested from panel .updateTracks like this: 1. loader sends signal 'addTrack' -> 2. panel.updateTracks received the signal and sends 'loadBindingList' -> 3. loader.loadBindingList received the signal and sends 'loadBindingList' and sends updateTracks -> 2.... Infinite loop. However, we can solve this by adding a function like panel.addTracks() { sends signal 'loadBindingList' }. I don't think it's necessary.